src/V3TraceDecl.cpp

```diff
@@ -583,7 +581,8 @@ class TraceDeclVisitor final : public VNVisitor {
         if (pair.second) {
             string dtypeName = skipTypep->cDTypeName();
             const string name{"trace_init_dtype__" + dtypeName};
-            // NOCOMMIT -- should we only use V3UniqueNames instead of worrying about cDTypeName()?
+            // NOCOMMIT  -- should we only use V3UniqueNames instead of worrying about
+            // cDTypeName()?
```

Comment:
I cooked up `cDTypeName()` when I thought the stringified version of a type name would be sufficiently unique.  But of course, two different scope may have the same type name for different types.  Do you think it's worthwhile to have a human readable version of the type name in these function names?  Or should I just rely solely on V3UniqueNames and get rid of `cDTypeName`?